hera_corr_cm/hera_corr_cm.py

```diff
@@ -427,7 +474,7 @@ def get_hashpipe_status(self):
                     fields = {k: float(vals[k])}
                     if np.isnan(fields[k]):
                         continue
-                except:
+                except:  # noqa
```

Comment:
bare excepts aren't great.

Reply:
I'll remove noqa and add issue per above.